CHANGELOG.md

```diff
@@ -161,6 +161,7 @@ More information on using Ransack can be found from the [Ransack documentation](
 
 ### Removed
 
+- **decidim-core**: The `rectify` gem has been removed from the stack as of [\#9101](https://github.com/decidim/decidim/pull/9101). If you are a library developer, replace any `Rectify::Query` to `Decidim::Query`, replace any `Rectify::Command` with `Decidim::Command`. Replace any `Rectify::Presenter` with `SimpleDelegator` (Already deprecated in [\#8758](https://github.com/decidim/decidim/pull/8758))
```

Comment:
> If you are a library developer, replace any `Rectify::Query` to `Decidim::Query`

If you are a library developer, replace any `Rectify::Query` **with** `Decidim::Query`